src/app/(home)/layout.tsx

```diff
@@ -12,6 +12,7 @@ export default async function HomeLayout({
 }) {
   return (
     <>
+      {/* @ts-expect-error RSC */}
```

Comment:
here has some [solution](https://github.com/microsoft/TypeScript/pull/51328), but IDK how to😁

***
get from [this](https://github.com/vercel/next.js/issues/43537)